MongoProject/src/main/java/com/example/demo/service/IUserService.java

```diff
@@ -5,8 +5,13 @@
 import org.springframework.stereotype.Service;
 
 import com.example.demo.model.User;
+import com.example.demo.model.response.UserResponse;
 @Service
 public interface IUserService {
 	List<User> findPaginated(int pageno,int pagesize);
-
+	public UserResponse createUser(UserResponse userResponse);

```

Comment:
rename UserResponse to UserDTO and extend it to create UserResponse which will have link too.